File: 160715/file/ruby/five.rb
# 5. 实现一个方法，传入一个文件名filename，创建他，停留5秒后，又删除它
file_name = "test1"

def create_file_and_delete(file_name)
  create_temp_file = File.new("#{file_name}.txt", 'w')
  sleep 5
  File.delete("#{file_name}.txt")
end

create_file_and_delete(file_name)

File: 160715/file/java/five.java
// 5. 实现一个方法，传入一个文件名filename，创建他，停留5秒后，又删除它
import java.io.File; 
import java.lang.InterruptedException;
import java.io.IOException;

public class five{
  public static void main(String[] args) {
    String filename = "text";
    createFileSleepAndDelete(filename);
  }

  public static void createFileSleepAndDelete(String filename){
    try{
      File file = new File(filename + ".txt");
      file.createNewFile();
      Thread.sleep(5000);
      file.delete();
    }catch(InterruptedException e){
      e.printStackTrace();
    }catch(IOException exception){
      exception.printStackTrace();
    }

  }
}